modules/bvl_feedback/test/TestPlan.md

```diff
@@ -1,10 +1,17 @@
 # Behavioural Feedback Test Plan
 
-1. Behavioural feedback button (notepad in the toolbar) should show up on the following pages:
- * Candidate Profile
- * Instrument List
- * Any instrument
-[Automation Test]
+1. Behavioural feedback button should show up on the following pages:
+ * Candidate Profile  
+ To find the Candiadate profile, first navigate to 'Access Profile', then select the specific Candidate profile from the list.  
+ ![image](images/candidate_list.PNG)  
+ * Instrument List  
+ To find an instrument, first navigate to a specific Candidate Profile, then click on a specific Visit Label.  
+ ![image](images/visit_label.PNG)  
+ A list of Intruments shall be found in this page as shown below.  
+ ![image](images/instrument_list.PNG)  
+ * Any instrument [Automation Test]   
+ *Note: The feedback button has the notepad icon in the toolbar as shown below.*  
```

Comment:
across loris 
```suggestion
 *Note: The feedback button is represented by the notepad icon in the menu as shown below.*  
```
* Menu (not toolbar)

Reply:
i think you should move this to the first line of your modifications - this pencil/notepad icon is the same trigger across multiple modules, not just in the instrument form